dotnet/src/SemanticKernel.Abstractions/Contents/ChatMessageContent.cs

```diff
@@ -33,6 +34,12 @@ public class ChatMessageContent : KernelContent
     [JsonIgnore]
     public Encoding Encoding { get; set; }
 
+    /// <summary>
+    /// The source of the message that generated it.
+    /// </summary>
+    [Experimental("SKEXP0101")]
+    public object? Source { get; set; }
```

Comment:
Unit test for serialization is failing because this property is not included in expected result:
![image](https://github.com/microsoft/semantic-kernel/assets/13853051/730ce076-04d4-4328-baac-f5f30c2d2a56)

Since it's going to be source instance which can be anything, should we put `[JsonIgnore]` attribute on this property?

Reply:
excluded for now.